### DISCUSSION TITLE
NeuralProphet install does not work in version 3.7.15

### DISCUSSION THREAD

**Discussion by Dalmasi11** [ourownstory/neural_prophet#872] [Q&A - get help using NeuralProphet] [answered]
Basically, the title says it all. When installing, I get the following error:

`ImportError: cannot import name 'Literal' from 'typing' (/usr/lib/python3.7/typing.py)`

It worked fine until yesterday. What did change? I am aware taht Literal requires version 3.8, may that be the problem?

**Comment by ourownstory** [comment, marked as answer]
@Dalmasi11 
Sorry for causing the error - it should be fixed in #869

Please open an issue if it persists (after re-installing NeuralProphet).

**Reply by Dalmasi11** [reply]
Yes, I managed to solve it, thanks! It worked after installing again.
